Cargo.toml

```diff
@@ -46,17 +46,19 @@ log = "0.4"
 regex = "1"
 serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.8"
-tox = "0.1.1"
+crypto_box = "0.7"
+rand = "0.8"
+tox = { git = "https://github.com/tox-rs/tox.git" }
```

Comment:
I granted ownership of tox* crates on crates.io to you.

Reply:
Thanks, but we can't make a release with this version yet. It has a deadlock that happens quite often when running DHT server, so we have to fix it first.

Reply:
I see